lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.rb

```diff
@@ -75,7 +75,8 @@ def search_image(image_name)
     when 0
       fail "Image #{image_name} not found!"
     else
-      fail "More than one image found for #{image_name}!"
+      log.debug "More than one image found for #{image_name}!"
```

Comment:
It used to check that there was exactly one match. However, official images don't have a "username" prefix, so a search for one of these almost always returns more than one. In either case, we "pull" the exact name, not the first match. This check is really only to prevent attempting to pull a non-existent image.